scripts/buck_setup.sh

```diff
@@ -14,19 +14,19 @@ python3 generate-xnnpack-wrappers.py
 printf "\n[Downloading bazel-skylib-1.0.2]\n"
 curl -L -o /tmp/bazel-skylib-1.0.2.tar.gz https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz
 mkdir bazel-skylib
-rm -r bazel-skylib/
+rm -rf bazel-skylib/
 tar -xf /tmp/bazel-skylib-1.0.2.tar.gz -C bazel-skylib/
 
 # glog
 printf "\n[Downloading glog-0.4.0]\n"
 curl -L -o /tmp/glog-0.4.0.tar.gz https://github.com/google/glog/archive/v0.4.0.tar.gz
 tar -xf /tmp/glog-0.4.0.tar.gz -C /tmp/
-rm -r glog/
+rm -rf glog/
```

Comment:
Instead of deleting, maybe just `mv -f`?

Reply:
if the target folder already exists, it will come to a subfolder inside it

Reply:
Something like that:
```
mkdir -p glog; curl -L https://github.com/google/glog/archive/v0.4.0.tar.gz|tar zx -C glog --strip-components 1
```
(let me propose the PR that addresses it)

Reply:
https://github.com/pytorch/pytorch/pull/80198